### PR TITLE
Setting up the project with laravel ui

### PR DESCRIPTION
**Pull request by SamZuckerberg**

### CODE REVIEW

File: app/Http/Controllers/Auth/ConfirmPasswordController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    |
+    | This controller is responsible for handling password confirmations and
+    | uses a simple trait to include the behavior. You're free to explore
+    | this trait and override any functions that require customization.
+    |
+    */
+
+    use ConfirmsPasswords;
+
+    /**
```
Comment (Gabriellsf9) [inline app/Http/Controllers/Auth/ConfirmPasswordController.php:24]: very nice BRO

**Review by Gabriellsf9 — APPROVED** [review]

File: app/Http/Controllers/Auth/RegisterController.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    */
+
+    use RegistersUsers;
+
+    /**
+     * Where to redirect users after registration.
+     *
+     * @var string
+     */
+    protected $redirectTo = RouteServiceProvider::HOME;
```
Comment (JulianoRaimo) [inline app/Http/Controllers/Auth/RegisterController.php:32]: ooowwwww heee

**Review by JulianoRaimo — APPROVED** [review]

File: app/Http/Controllers/Auth/ConfirmPasswordController.php (hunk 1 above)
Comment (SamZuckerberg) [reply app/Http/Controllers/Auth/ConfirmPasswordController.php:24]: image

File: app/Http/Controllers/Auth/RegisterController.php (hunk 2 above)
Comment (SamZuckerberg) [reply app/Http/Controllers/Auth/RegisterController.php:32]: image

File: app/Http/Controllers/Auth/ConfirmPasswordController.php (hunk 1 above)
Comment (Gabriellsf9) [reply app/Http/Controllers/Auth/ConfirmPasswordController.php:24]: this is Juliano?

Comment (JulianoRaimo) [reply app/Http/Controllers/Auth/ConfirmPasswordController.php:24]: no its you 🗡️

**Pull request merged**